pkg/cli/project.go

```diff
@@ -80,12 +84,49 @@ func (a *Project) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	out := table.NewWriter(tables.ProjectClient, a.Quiet, a.Output)
-	for _, project := range projectNames {
-		out.Write(projectEntry{
-			Name:    project,
-			Default: defaultProject == project,
-		})
+
+	var wg sync.WaitGroup
```

Comment:
Just to make sure I'm understanding - we're doing this asynchronously so that we can process them all faster, right?